package.json

```diff
@@ -42,6 +42,7 @@
     "@types/cross-spawn": "6.0.2",
     "@types/fs-extra": "9.0.13",
     "@types/jsdom": "16.2.14",
+    "@types/mocha": "9.1.1",
```

Comment:
why is that needed? 🤔

Reply:
Because we currently don't have any types and the whole editor is red.
This is not directly related to the ticket, but made my life easier.